docs/community-resources.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Community Resources
+articles:
+  [
+    {
+      title: "TkDodo's Blog Posts",
+      url: 'https://tkdodo.eu/blog/tags/react-query',
+      description: 'TkDodo, a maintainer of TanStack Query, writes a series of blog posts about the library. These articles offer general best practices and often present opinionated perspectives on using TanStack Query.',
+    },
```

Comment:
we have a lot of links to the blog pages from the guides, e.g.:

https://tanstack.com/query/v5/docs/framework/react/typescript#further-reading

They don’t link to the blog directly, but they link to the community resources. We would need to change them to point directly to the blog articles.